### ISSUE TITLE
__version__ attribute

### ISSUE THREAD

**Issue by clach04** [ActianCorp/sqlalchemy-ingres#10]
Add a `__version__` to the top level module and/or `IngresDialect` class.

No other dialects do this but would make it easier to introspect the dialect directly (than system metadata).

**Comment by hab6** [comment]
Added via https://github.com/ActianCorp/ingres_sa_dialect/pull/18

**Comment by clach04** [comment]
I don't see where this was implemented/added, please link to line number before closing.

**Comment by hab6** [comment]
Possibly implement similar to [example](https://github.com/clach04/x10_any/blob/313b0423e2022247e7b5339c888a8201be0277f2/setup.py#L30).

**Comment by hab6** [comment]
Verifying this works as implemented via #25.

```
(.venv) C:\work\saDialect\gh10>python -V
Python 3.10.7

(.venv) C:\work\saDialect\gh10>pip show sqlalchemy-ingres
Name: sqlalchemy-ingres
Version: 0.0.4.dev1
Summary: SQLAlchemy dialect for Actian databases; Actian Data Platform (nee Avalanche), Actian X, Ingres, and Vector
Home-page: https://github.com/ActianCorp/sqlalchemy-ingres
Author: Chris Clark
Author-email: Chris.Clark@actian.com
License: Apache-2.0
Location: c:\work\sadialect\gh10\.venv\lib\site-packages
Requires: sqlalchemy
Required-by:

(.venv) C:\work\saDialect\gh10>python
Python 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import sqlalchemy_ingres
>>> print(sqlalchemy_ingres.__version__)
0.0.4.dev1
>>> quit()

(.venv) C:\work\saDialect\gh10>
```

@clach04 Is this okay to close?
